torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -466,6 +470,16 @@ def fold_weight(
                 )
                 packed_weight = prepacking_module()
                 packed_weights[node.name] = packed_weight
+                if keep_original_weights:
+                    original_weights = list(prepacking_module.state_dict().values())
+                    original_weights_lookup[str(lookup_counter)] = sorted(
+                        original_weights, key=lambda x: x.numel(), reverse=True
+                    )
+                    if len(original_weights_lookup[str(lookup_counter)]) == 1:
+                        # bias is None
+                        original_weights_lookup[str(lookup_counter)].append(None)
+                    lookup_counter += 1
+    lookup_counter = 0
```

Comment:
we have some utils for this actually

Reply:
this allows people to create some attribute with some prefix, and makes sure there is no conflict, see how it is used here: https://github.com/pytorch/pytorch/blob/1598d458797e69376a9a148bd37fb6e8167d22e3/torch/ao/quantization/fx/prepare.py#L496-L497

Reply:
Thanks! I am using lookup_counter temporarily, and updating the real key_name in line 514 with deleting lookup_counter afterwards